cg/apps/gt.py

```diff
@@ -37,3 +46,31 @@ def upload(self, bcf_path: str, samples_sex: dict, force: bool=False):
                 analysis_obj.sex = samples_sex[analysis_obj.sample_id]['analysis']
                 analysis_obj.sample.sex = samples_sex[analysis_obj.sample_id]['pedigree']
                 self.commit()
+
+    def get_trending(self, sample_id: str = '', days: str = '') -> dict:
```

Comment:
Looks like the return value is a byte string, not a dict 

https://docs.python.org/2/library/subprocess.html#subprocess.check_output

Reply:
correct